nexus/src/app/background/tasks/support_bundle_collector.rs

```diff
@@ -494,8 +494,12 @@ impl BundleCollection {
 
         // Create the zipfile as a temporary file
         let mut zipfile = tokio::fs::File::from_std(bundle_to_zipfile(&dir)?);
+        let total_len = zipfile.metadata().await?.len();
```

Comment:
By the way John pointed this out on one of my previous PRs but you can SeekFrom::End(0) (since we later seek back to the beginning on line 503) to get the length without having to do an extra stat call. Either way works, just thought I would mention it.

Reply:
I think this will end up being roughly equivalent; it's a "stat now, seek later" or "seek now, seek later", right? Either way, we need a syscall to get the length